### PR TITLE
Sync to ntc cookiecutter development standards

### PR DESCRIPTION
**Pull request by itdependsnetworks**
Sync to ntc cookiecutter development standards

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -113,7 +113,18 @@ jobs:
       matrix:
         python-version: ["3.7", "3.8", "3.9", "3.10"]
         db-backend: ["postgresql"]
-        nautobot-version: ["1.3.3", "stable"]
+        nautobot-version: ["stable"]
```
Comment (whitej6) [inline .github/workflows/ci.yml:116]: Feel like this is unneeded based on above matrix

File: development/creds.example.env [hunk 2]
```diff
@@ -1,11 +1,27 @@
...
-REDIS_PASSWORD=${NAUTOBOT_REDIS_PASSWORD}
 
+# MySQL Credentials
+MYSQL_ROOT_PASSWORD=${NAUTOBOT_DB_PASSWORD}
+MYSQL_PASSWORD=${NAUTOBOT_DB_PASSWORD}
+
+# Use these to override values in development.env
+# NAUTOBOT_DB_HOST=localhost
+# NAUTOBOT_REDIS_HOST=localhost
+# NAUTOBOT_CONFIG=development/nautobot_config.py
```
Comment (whitej6) [inline development/creds.example.env:27]:
```suggestion
# NAUTOBOT_CONFIG=development/nautobot_config.py

```

File: development/development.env [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+# NAUTOBOT_DB_PORT=5432
+
+# Needed for Postgres should match the values for Nautobot above
+POSTGRES_USER=${NAUTOBOT_DB_USER}
+POSTGRES_DB=${NAUTOBOT_DB_NAME}
+
+# Needed for MYSQL should match the values for Nautobot above
+MYSQL_USER=${NAUTOBOT_DB_USER}
+MYSQL_DATABASE=${NAUTOBOT_DB_NAME}
+MYSQL_ROOT_HOST=%
```
Comment (whitej6) [inline development/development.env:40]:
```suggestion
MYSQL_ROOT_HOST=%

```

File: development/development_mysql.env [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# Custom ENVs for Mysql
+# Due to docker image limitations for Mysql, we need "root" user to create more than one database table
+NAUTOBOT_DB_USER=root
+MYSQL_USER=${NAUTOBOT_DB_USER}
```
Comment (whitej6) [inline development/development_mysql.env:4]:
```suggestion
MYSQL_USER=${NAUTOBOT_DB_USER}

```

File: development/docker-compose.postgres.yml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+      - "creds.env"
+    volumes:
+      # - "./nautobot.sql:/tmp/nautobot.sql"
+      - "postgres_data:/var/lib/postgresql/data"
+    healthcheck:
+      test: "pg_isready --username=$$POSTGRES_USER --dbname=$$POSTGRES_DB"
+      interval: "10s"
+      timeout: "5s"
+      retries: 10
+    ports:
```
Comment (whitej6) [inline development/docker-compose.postgres.yml:21]: Default port exposures for DBs & Redis should be commented out

File: invoke.example.yml [hunk 6]
```diff
@@ -1,9 +1,9 @@
 ---
 nautobot_firewall_models:
-  project_name: "nautobot-firewall-model"
-  nautobot_ver: "1.1.0"
+  project_name: "nautobot_firewall_models"
+  nautobot_ver: "1.3.5"
```
Comment (whitej6) [inline invoke.example.yml:4]: Good catch but doesn't factor in I believe, should be removed. Similar to issue 219 for internal standards

File: pyproject.toml [hunk 7]
```diff
@@ -96,4 +115,4 @@ build-backend = "poetry.core.masonry.api"
 testpaths = [
     "tests"
 ]
-addopts = "-vv --doctest-modules"
+addopts = "-vv --doctest-modules"
```
Comment (whitej6) [inline pyproject.toml:118]:
```suggestion
addopts = "-vv --doctest-modules"

```

**Review by whitej6 — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (itdependsnetworks) [reply .github/workflows/ci.yml:116]: This was used to limit the matrix, not expand

File: invoke.example.yml (hunk 6 above)
Comment (itdependsnetworks) [reply invoke.example.yml:4]: @whitej6 to take care of this.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (itdependsnetworks) [reply .github/workflows/ci.yml:116]: added comment per @whitej6

**Review by whitej6 — APPROVED** [review]

**Pull request merged**